README.md

```diff
@@ -2,6 +2,8 @@
 
 A project based learning activity for people who are getting started with Git and GitHub.
 
+### Instructions for playing the game
+
```

Comment:
locate bricks in correct position without leaving empty spaces in between bricks